example/ios/VisionCameraPluginBarcodeScannerExample-Bridging-Header.h

```diff
@@ -1,3 +1,6 @@
 //
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
+
+#import <VisionCamera/FrameProcessorPlugin.h>
+#import <VisionCamera/Frame.h>
```

Comment:
is this a required step to get this plugin working?

Reply:
I don't believe so, I was simply matching the example v2 plugin as close to possible to eliminate variables.